### PR TITLE
KDE updates

### PR DESCRIPTION
**Pull request by dilyn-corner**
Big changes in the KISS-kde repo necessitate broad updates to the wiki.

### CODE REVIEW

File: desktops/kde.txt [hunk 1]
```diff
@@ -130,28 +284,55 @@ if you know you will not need bluetooth, you have no need for bluedevil.
...
-    |          -O qt5\#$qtver.tar.gz                                           |
+    |          -O qt5@$qtver.tar.gz                                            |
     |   $ wget $url/qt5-declarative.$qtver.tar.gz \                            |
-    |         -O qt5-declarative\#$qtver.tar.gz                                |
+    |         -O qt5-declarative@$qtver.tar.gz                                 |
+    |                                                                          |
+    |   # Not a build-time requirement, but useful for e.g. falkon             |
     |   $ wget $url/qt5-webengine.$qtver.tar.gz \                              |
-    |          -O qt5-webengine\#$qtver.tar.gz                                 |
+    |          -O qt5-webengine@$qtver.tar.gz                                  |
```
Comment (dylanaraps) [inline desktops/kde.txt:314]: `@` as a separator is not yet in a full package manager release. To speed things up I may backport some things to a new 5.X.X release.

**Comment by dylanaraps** [comment]
This is amazing stuff. :+1:

File: desktops/kde.txt (hunk 1 above)
Comment (dilyn-corner) [reply desktops/kde.txt:314]: I'm just being ready for the change :)

**Comment by dilyn-corner** [comment]
The most striking changes here are the optionality of nonkiss things like `polkit`. Some of the build scripts had to be tweaked for the options available but I think the solution is reasonable. 

Additionally impressed with the low idle resource usage! With `sowm` the desktop issues at just shy of 300MB used - roughly half what `kwin` uses. 

There are just a few grammar issues and some phrasing changes I'd like to make before I think it's ready. If you think `kiss` will be updated to 6 before the end of the month we can hold off on the merge until then, or I can just change it if you'd like to get this in soon.

**Comment by dilyn-corner** [comment]
Just your friendly neighborhood bump for a final review :)

File: desktops/kde.txt [hunk 2]
```diff
@@ -48,110 +58,291 @@ ________________________________________________________________________________
...
+    |   # If you opt to install e.g. elogind                                   |
+    |   $ kiss b coreutils && kiss i coreutils                                 |
+    |   $ kiss a coreutils /usr/bin/realpath                                   |
+    |   $ kiss a coreutils /usr/bin/mktemp                                     |
+    |   $ kiss a coreutils /usr/bin/ln                                         |
+    |                                                                          |
     +--------------------------------------------------------------------------+
 
+    NOTE: If you had packages like xorg-server installed, you might want to 
+    rebuild them to pickup on the new device manager they can take advantage of.
```
Comment (dylanaraps) [inline desktops/kde.txt:289]: `pickup` -> `pick up` ?

**Comment by dylanaraps** [comment]
I'm very happy with how this turned out. I have no other comments. :+1:

The last blocker for this PR is the new release of the package manager (for '<pkg>@<ver>` naming scheme). I have plenty of time now as I'll be in lockdown for a month as of tomorrow. Once 6.0.0 is released (I may also decide to cherry-pick the important bug fixes / features for a 5.X.X, we'll see).

File: desktops/kde.txt (hunk 1 above)
Comment (dylanaraps) [reply desktops/kde.txt:324]: Change has been made.

**Comment by dilyn-corner** [comment]
Had a typo in all my bootloader entries and my laptop has been a brick for the last week (: If it good for you, it's good for me! Merge away.

**Pull request merged**